dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst/csharp/MacCatalystApp1.csproj

```diff
@@ -1,11 +1,33 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <TargetFramework>net7.0-maccatalyst</TargetFramework>
-    <RuntimeIdentifier>maccatalyst-x64</RuntimeIdentifier>
+  <PropertyGroup Condition="$(TargetFramework.Contains('-maccatalyst')) and '$(Configuration)' == 'Debug'">
+		<MtouchLink>none</MtouchLink>
+		<EnableCodeSigning>false</EnableCodeSigning>
+		<ProvisionType>Manual</ProvisionType>
+		<CreatePackage>false</CreatePackage>
+		<EnablePackageSigning>true</EnablePackageSigning>
+		<CodesignKey>Apple Distribution: YOURNAME (*******)</CodesignKey>
+		<CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
+		<CodesignProvision>YOUR PROFILE NAME</CodesignProvision>
+		<PackageSigningKey>3rd Party Mac Developer Installer: YOURNAME (*******)</PackageSigningKey>
```

Comment:
Will having these properties with the dummy values cause issues before the user changes them? Would adding a comment reminding the users to change these values be overkill?

Reply:
That's a good point, I commented these blocks out and made a reminder comment to replace the placeholder text.